resources/files/windows/bolt.bat

```diff
@@ -0,0 +1,6 @@
+@echo off
+SETLOCAL
+
+call "%~dp0environment.bat" %0 %*

```

Comment:
I'm not sure there's much point to separating environment.bat out yet, but also no reason not to.